ipwhois/net.py

```diff
@@ -111,6 +111,10 @@ def __init__(self, address, timeout=5, proxy_opener=None):
         # Default timeout for socket connections.
         self.timeout = timeout
 
+        self.resolver = dns.resolver.Resolver()
```

Comment:
let's change the naming of self.resolver to self.dns_resolver to avoid confusion